code/modules/events/immovable_rod.dm

```diff
@@ -11,7 +11,7 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 	announceWhen = 5
 
 /datum/event/immovable_rod/announce()
-	event_announcement.Announce("What the fuck was that?!", "General Alert")
+	event_announcement.Announce("Unidentified object inbound. Brace for impact.", "Collision Alert")
```

Comment:
I don't mind it myself but it might break some people's immurshuns since "CC wouldn't swear" or "it's too unprofessional" or whatever